packages/core/android/src/main/java/com/segmentanalyticsreactnative/AnalyticsReactNativeModule.kt

```diff
@@ -218,6 +225,24 @@ class AnalyticsReactNativeModule : ReactContextBaseJavaModule, ActivityEventList
     sovran?.dispatch("add-deepLink-data", properties)
   }
 
+    fun setAnonymousId(anonymousId: String) {
+     val properties = Hashtable<String, String>()
+     properties["anonymousId"] = anonymousId
+
+      if (currentActivity == null) {
+        Log.d(name, "No activity found")
+        return
+      }
```

Comment:
I know you were getting an error about this before but this means that if the activity is not found it is going to drop all the calls.

Did you confirm it is actually sending the anonymousId before RN startup after this change? Or why activity was false?